### DISCUSSION TITLE
How to apply Texture for Shape?

### DISCUSSION THREAD

**Discussion by hlvlad** [molstar/molstar#191] [Q&A]
Hi, is there a way to apply texture for Shape instead of uniform color? I'm interested in 1D texture (stripes).

**Comment by dsehnal** [comment]
Hi, using a palette in the theme should do the trick: https://github.com/molstar/molstar/blob/master/src/mol-theme/color.ts#L52

Basically you specify a set of colors and the color returned by the theme then serves as a coordinate to 1D texture that corresponds to the palette.

Here's example usage: https://github.com/molstar/molstar/blob/master/src/examples/basic-wrapper/custom-theme.ts#L9
